DicomRtImportExport/Logic/vtkSlicerDicomRtReader.h

```diff
@@ -88,6 +88,9 @@ class VTK_SLICER_DICOMRTIMPORTEXPORT_LOGIC_EXPORT vtkSlicerDicomRtReader : publi
   /// Get name of beam
   const char* GetBeamName(unsigned int beamNumber);
 
+  /// Get type of beam
+  const char* GetBeamType(unsigned int beamNumber);
+
   /// Get beam isocenter for a given beam
   double* GetBeamIsocenterPositionRas(unsigned int beamNumber); // DEPRICATED

```

Comment:
Could you also fix this typo? It's spelled deprecated. Thanks!

Reply:
fixed